__tests__/overview.puppeteer.test.js

```diff
@@ -0,0 +1,40 @@
+import { render, screen, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import 'regenerator-runtime/runtime';
+import puppeteer from 'puppeteer';
```

Comment:
Puppeteer is imported, but seems like it is not used in anywhere

Reply:
Good catch, I will remove

Reply:
Well I thought I had ignored the commit of this file, but apparently I need to review how to add files to a gitignore